CHANGELOG.md

```diff
@@ -5,8 +5,13 @@
   - Adds `#[callback_unwrap]` to replace `callback`
 - mock: Update `method_names` field of `AddKeyWithFunctionCall` to a `Vec<String>` from `Vec<Vec<u8>>`. [PR 555](https://github.com/near/near-sdk-rs/pull/555)
   - Method names were changed to be strings in `4.0.0-pre.2` but this one was missed
-- env: Update the register used for temporary `env` methods to `u64::MAX - 2` from `0`. [PR 557](https://github.com/near/near-sdk-rs/pull/489).
+- env: Update the register used for temporary `env` methods to `u64::MAX - 2` from `0`. [PR 557](https://github.com/near/near-sdk-rs/pull/557).
   - When mixing using `sys` and `env`, reduces chance of collision for using `0`
+- store: Implement caching `LookupMap` type. This is the new iteration of the previous version of `near_sdk::collections::LookupMap` that has an updated API, and is located at `near_sdk::store::LookupMap`. [PR 487](https://github.com/near/near-sdk-rs/pull/487).
```

Comment:
well both do, the only difference is that unordered map keeps track of keys in this vector like structure so that it can be iterated (`LookupMap` isn't aware of what elements exist in it). I'll try to make docs more clear